uvod-do-programovani-2/slicing-metody-moduly/excs/prumer.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Průměr
+demand: 2
+---
+
+Mějme proměnnou `s`, ve které předpokládáme uložený nějaký seznam. Sestavte v Python konzoli výraz (vzoreček), který spočítá průměrnou hodnotu v takovém seznamu. Otestujte jej na seznamech různých délek.
```

Comment:
Počítáme v této kapitole s prací v konzoli, nebo může být zadání obecné (Sestavte výraz ...)?